stdlib/ballerina-http/src/main/ballerina/http/annotation.bal

```diff
@@ -55,29 +55,34 @@ public struct CorsConfig {
 
 @Description {value:"Initializes the CorsConfig struct with default values."}
 @Param {value:"config: The CorsConfig struct to be initialized"}
-public function <CorsConfig config> CorsConfig() {
+public function <CorsConfig config> CorsConfig () {
     config.maxAge = -1;
 }
 
 
 @Description {value:"Configuration for a WebSocket service."}
-@Field {value: "endpoints: An array of endpoints the service would be attached to"}
+@Field {value:"endpoints: An array of endpoints the service would be attached to"}
 @Field {value:"basePath: Path of the WebSocket service"}
 @Field {value:"subProtocols: Negotiable sub protocol by the service"}
 @Field {value:"idleTimeoutInSeconds: Idle timeout for the client connection. This can be triggered by putting onIdleTimeout resource in WS service."}
-public struct WSServiceConfig {
-    ServiceEndpoint[] endpoints;
-    WebSocketEndpoint[] webSocketEndpoints;
-    string basePath;
-    string[] subProtocols;
-    int idleTimeoutInSeconds;
+public type WSServiceConfig object {
+    public {
+        ServiceEndpoint[] endpoints;
+        WebSocketEndpoint[] webSocketEndpoints;
```

Comment:
AFAIU we do not need serviceEndpoint and websocketEndpoint inside the annotation.